src/Views/Activity/MyDayOffline.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @class crm.Views.Offline.List
+ *
+ * @extends argos._ListBase
+ * @requires argos._ListBase
+ *
+ *
+ */
+import declare from 'dojo/_base/declare';
+import OfflineList from '../Offline/List';
+// import _MetricListMixin from '../_MetricListMixin';
```

Comment:
Yes I 'm not finished with it yet I will need to add back in the metric back in for the offline counts I was just seeing if it would merge